pkg/daemons/config/types.go

```diff
@@ -104,7 +104,28 @@ type Agent struct {
 	EnableIPv6              bool
 }
 
+// ControlQuorumArgs contains parameters that all control plane nodes in HA must share
+type ControlQuorumArgs struct {
```

Comment:
I'm not sure calling these Quorum arguments conveys the right message; quorum suggests that they only need to agree across a voting quantity or minimum number of servers. In fact they must be the same on ALL servers.

```suggestion
type CriticalControlArgs struct {
```

Reply:
Agree